manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -25,6 +25,8 @@ webhooks:
         path: "/inject"
       {{- end }}      
       caBundle: {{ .Values.sidecarInjectorWebhook.caBundle }}
+    sideEffects: None
```

Comment:
Can you comment on why the sideEffects is none for MutatingWebhookConfiguration?